### ISSUE TITLE
Nitropy Doesn't Terminate When Executed to Export a Certificate

### ISSUE THREAD

**Issue by nitro-rico** [Nitrokey/pynitrokey#228]
nitropy nethsm --host $NETHSM_HOST --username Operator get-key myFirstKey --public-key > public.pem 

- Certificate is exported
- Requires Ctrl-C to terminate
- Otherwise it hangs

**Comment by robin-nitrokey** [comment]
As you do not specify the password on the command line, nitropy shows a password prompt.  As you redirect the output to a file, the prompt is written to that file but nitropy still waits for the password input.  We could print the prompt to stderr instead to avoid this.  @szszszsz Thoughts?

**Comment by szszszsz** [comment]
> We could print the prompt to stderr instead to avoid this.

Yes, I highly recommend that. This should be the standard behavior across pynitrokey, so that only the actual results are printed to stdout for the scripting.

**Comment by robin-nitrokey** [comment]
We currently have three ways to issue prompts: 1) `click.prompt`/`click.confirm`, 2) setting `prompt=True` for a `click` option and 3) `pynitrokey.helpers.AskUser`.  This can be easily fixed for 1) by setting `err=True` and for 3) by modifying `AskUser.get_input`.  For 2), we have to create a helper function that can be used as a callback with the `default` option as `click` only supports stdout with `prompt=True`.  That’s not really nice, but I think we can’t work around it.

Also, for 1), I’d like to create helper functions that set `err=True` so that we don’t have to add that to every invocation.  We could also use `AskUser` instead, but that would require adding some more functionality to it (e. g. default values).
